src/Components/Modals/ModalPhotoDisplay/ModalPhotoDisplay.css

```diff
@@ -1,53 +1,73 @@
-.Modal_photos{
-    background-color: white;
-    position: absolute;
-    top: 30%;
-    left: 34%;
-    z-index: 12;
-    width: 600px;
-    height: 400px;
-    border-radius: 5px;
-}
-
-.currentClickedImage {
-    width: 400px;
+.modalPhotoHolder {
+    position: relative;
+    top: 15%;
+    left: 20%;
+    width: 800px;
     height: 400px;
-    padding-left: 40px;
-    padding-right: 40px;
-}
 
-.arrowIcons {
-    font-size: 40px;
-    color: gray;
-    display: inline-block;
-    margin-bottom: 200px;
-}
-
-.arrowIcons:hover {
-    cursor: pointer;
 }
 
 .thumbParentHolder {
     position: absolute;
-    top: 22.4%;
-    left: 34%;
     z-index: 12;
-    width: 600px;
+    width: 650px;
     overflow: scroll;
 }
 
 .thumbHolder {
     width: 650px;
     height: 60px;
-    background-color: rgb(216, 212, 212);
+    margin: 0 auto;
 }
 
 .thumbs {
     width: 50px;
     height: 50px;
     margin-right: 5px;
+    border-radius: 10px;
 }
 
 .currentClickedImageThumb {
-    border: 2px solid rgb(0, 255, 76);
+    border: 2px solid #F4A460;
 }
+
+.modal_photos{
+    z-index: 12;
+    position: absolute;
+    top: 70px;
+    border-radius: 5px;
+}
+
+.currentClickedImage {
+    max-height: 85%;
+    max-width: 85%;
+    margin: 0 auto;
+    border-radius: 10px;
+}
+
+.arrowIcons {
+    font-size: 40px;
+    color: rgb(87, 87, 87);
+    margin-top: 200px;
+    background: rgba(255, 255, 255, 0.746);
+    border-radius: 50%;
+    width: 40px;
+    height: 40px;
+    padding-bottom: 6px;
+}
+
+.arrowIcons:hover {
+    cursor: pointer;
+}
+
+.photoDisplay_iconAngleLeft {
```

Comment:
stop mixing camelcase and underscore!

Reply:
no need to fix, just stop for future lol

Reply:
ok 👯